openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -176,6 +176,10 @@ def load_from_module_store(cls, course_id):
                 course_overview = cls._create_from_course(course)
                 try:
                     course_overview.save()
+                    for tab in course.tabs:
+                        course_overview_tab = CourseOverviewTab.objects.create(
+                            tab_id=tab.tab_id, course_overview=course_overview)
+                        course_overview_tab.save()
```

Comment:
Please use [bulk_create](https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create) instead.  Otherwise, the database will be hit multiple times.